packages/application-shell/src/components/suspended-route/suspended-route.tsx

```diff
@@ -1,10 +1,12 @@
-import { Suspense } from 'react';
+import { Suspense, type ReactNode } from 'react';
 import { Route, type RouteProps } from 'react-router-dom';
 import LoadingSpinner from '@commercetools-uikit/loading-spinner';
 
 const SuspendedRoute = (props: RouteProps) => (
   <Route {...props}>
-    <Suspense fallback={<LoadingSpinner />}>{props.children}</Suspense>
+    <Suspense fallback={<LoadingSpinner />}>
+      {props.children as ReactNode}
```

Comment:
Do we actually need this casting?
It seems the `children` prop is already typed as a `ReactNode`:
```ts
children?: ((props: RouteChildrenProps<Params>) => React.ReactNode) | React.ReactNode | undefined;
```